lcdbackpack/lcdbackpack.py

```diff
@@ -234,6 +233,17 @@ def write(self, string):
 
         self._ser.write(str.encode(string))
 
+    def set_splash_screen(self, string, lcd_chars):
+        """
+        Sets the LCD splash screen. 
+        :param string: the text to be displayed at LCD start up
+        :param lcd_chars: the total characters of the LCD display
+        """
+        self._write_command([0x40])
+        self._ser.write('{{0: <{}}}'.format(lcd_chars).format(string).encode())
```

Comment:
I'll need to unpack exactly what this is doing! My python-fu is not so strong.

Reply:
This is mostly taken from: https://stackoverflow.com/a/5676884